srcpkgs/ktouch/template

```diff
@@ -0,0 +1,28 @@
+# Template file for 'ktouch'
+pkgname=ktouch
+version=18.12.1
+revision=1
+build_style=cmake
+hostmakedepends="extra-cmake-modules pkg-config"
+makedepends="libxkbfile-devel kdeclarative-devel kcmutils-devel
+qt5-xmlpatterns-devel qt5-tools-devel kio-devel qt5-plugin-mysql qt5-plugin-tds
+qt5-declarative-devel qt5-plugin-odbc qt5-plugin-pgsql qt5-plugin-sqlite
+qt5-quickcontrols2-devel"
+short_desc="Learn and practive touch typing"
+maintainer="1is7ac3 <isaac.qa13@gmail.com>"
+license="GPL-2.0-or-later"
+homepage="https://kde.org/applications/education/ktouch"
+distfiles="${KDE_SITE}/applications/${version}/src/${pkgname}-${version}.tar.xz"
+checksum=4529efa94dc0574790f492e9cc54ed898596c424f6444e025f81d0427a87746c
+
+pre_configure() {
+ _dirQml=$(which qmlcachegen-qt5)
```

Comment:
please don't modify the state of the system, instead edit the generated makefile to remove $XBPS_CROSS_BASE